
From: "Martin J. Bligh" <mbligh@aracnet.com>

From: Andy Whitcroft <apw@shadowen.org>

The comments for i386 allocate_pgdat indicate that the routine should be
modified to place the pgdat into node local memory.  However, this has
already been done as the pgdat is placed at node_remap_start_vaddr.  This
patch updates the comments to reflect this reality.

Signed-off-by: Andy Whitcroft <apw@shadowen.org>
Signed-off-by: Martin J. Bligh <mbligh@aracnet.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/mm/discontig.c |    7 +++++--
 1 files changed, 5 insertions(+), 2 deletions(-)

diff -puN arch/i386/mm/discontig.c~fix-allocate_pgdat-comments arch/i386/mm/discontig.c
--- 25/arch/i386/mm/discontig.c~fix-allocate_pgdat-comments	2004-06-22 11:11:15.734307968 -0700
+++ 25-akpm/arch/i386/mm/discontig.c	2004-06-22 11:11:15.742306752 -0700
@@ -129,8 +129,11 @@ static void __init find_max_pfn_node(int
 }
 
 /* 
- * Allocate memory for the pg_data_t via a crude pre-bootmem method
- * We ought to relocate these onto their own node later on during boot.
+ * Allocate memory for the pg_data_t for this node via a crude pre-bootmem
+ * method.  For node zero take this from the bottom of memory, for
+ * subsequent nodes place them at node_remap_start_vaddr which contains
+ * node local data in physically node local memory.  See setup_memory()
+ * for details.
  */
 static void __init allocate_pgdat(int nid)
 {
_
